queue_processors/autoreduction_processor/post_process_admin.py

```diff
@@ -188,17 +188,24 @@ def reduce(self):
                          self.message.serialize(limit_reduction_script=True))
             self.client.send(ACTIVEMQ_SETTINGS.reduction_started, self.message)
 
-            # Specify instrument directory
-            instrument_output_dir = MISC["ceph_directory"] % (self.instrument,
-                                                              self.proposal,
-                                                              self.run_number)
-
-            if self.instrument in MISC["excitation_instruments"]:
-                # Excitations would like to remove the run number folder at the end
-                instrument_output_dir = instrument_output_dir[:instrument_output_dir.rfind('/') + 1]
-
-            # Specify directories where autoreduction output will go
-            reduce_result_dir = MISC["temp_root_directory"] + instrument_output_dir
+            def specify_instrument_directories():
+                """Specifies instrument directories, including removal of run_number folder
+                if excitations instrument
+                :return (list) Directories where Autoreduction should output"""
+                # Specify instrument directory
+                instrument_output_dir = MISC["ceph_directory"] % (self.instrument,
+                                                                  self.proposal,
+                                                                  self.run_number)
+
+                if self.instrument in MISC["excitation_instruments"]:
+                    # Excitations would like to remove the run number folder at the end
+                    instrument_output_dir = instrument_output_dir[
```

Comment:
Can we assert that the run number folder exists before blindly removing it. As if someone does this elsewhere this will end up putting us into one parent higher than we expect